css/app/datepicker.css

```diff
@@ -89,7 +89,16 @@
 	border-left: 0;
 }
 
+.todayHilight {
+	background-color: rgb(255, 255, 170);
+}
+
+#datepicker table tbody tr.uib-weeks td:first-child button {
+	background-color: rgba(240, 240, 240, .9);
+}
+
 #datepicker table tbody tr.uib-weeks td:last-child button {
+	background-color: rgba(240, 240, 240, .9);
```

Comment:
I'm not sure this approach will work for both locales.

Maybe we have to add another class to the date picker element, that says whether the week starts on Sunday or Monday.